api/routes/states/openAPI.js

```diff
@@ -0,0 +1,85 @@
+const {
+  requiresAuth,
+  schema: { arrayOf, jsonResponse }
+} = require('../openAPI/helpers');
+
+const id = {
+  type: 'string',
+  description: '2-letter US State or Territory abbreviation, lowercase'
+};
+
+const name = {
+  type: 'string',
+  description: 'Full name of US State or Territory'
+};
+
+const medicaid_office = { // eslint-disable-line camelcase
+  type: 'object',
+  description: 'Contact information for this US State/Territory Medicaid office'
+};
+
+const stateSchema = {
+  type: 'object',
+  properties: {
+    id,
+    name
+  }
+};
+
+const fullStateSchema = {
+  type: 'object',
+  properties: {
+    id,
+    name,
+    medicaid_office
+  }
+};
+
+const getStates = {
+  '/states': {
+    get: {
+      tags: ['States'],
+      description: 'Get a list of all US States and Territories',
+      responses: {
+        200: {
+          description: 'List of US States and Territories',
+          content: jsonResponse(arrayOf(stateSchema))
+        }
+      }
+    }
+  }
+};
+
+const getState = {
+  '/states/{id}': {
+    get: {
+      tags: ['States'],
+      description: 'Get a single US State or Territiory',
+      parameters: [
+        {
+          name: 'id',
+          in: 'path',
+          description: id.description,
+          required: true,
+          schema: {
+            type: id.type
+          }
+        }
+      ],
+      responses: {
+        200: {
+          description: 'US State or Territory',
+          content: jsonResponse(fullStateSchema)
+        },
+        404: {
```

Comment:
should this be a 400 error?

Reply:
I was following this chart, but I'm open to following some other guidelines. https://restapitutorial.com/lessons/httpmethods.html

Reply:
I always think of 404 as a routing error. 400 is a Bad Request
https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html

Reply:
The `/users/{id}` endpoint returns 404 when supplied a bad id, while the `/apds/{id}` endpoint returns 400.  I suppose we should normalize on one or the other.